### PR TITLE
LOOP-1022: Revise Settings Guardrails Warning Copy

### PR DESCRIPTION
**Pull request by novalegra**
https://tidepool.atlassian.net/browse/LOOP-1022

### CODE REVIEW

File: LoopKit/TherapySetting.swift [hunk 1]
```diff
@@ -80,3 +80,42 @@ public extension TherapySetting {
         }
     }
 }
+
+// MARK: Guardrails
+public extension TherapySetting {
```
Comment (novalegra) [inline LoopKit/TherapySetting.swift:85]: I pulled out the copy into a centralized location so it's easier to follow/change

**Review by nhamming — APPROVED** [review]
LGTM

File: LoopKitUI/Views/Settings Editors/BasalRateScheduleEditor.swift [hunk 2]
```diff
@@ -139,6 +139,7 @@ private struct BasalRateGuardrailWarning: View {
         assert(!crossedThresholds.isEmpty)
 
         let caption = self.isZeroUnitRateSelectable && crossedThresholds.allSatisfy({ $0 == .minimum })
+            // ANNA TODO: ask MLee about this one
```
Comment (ginnyyadav) [inline LoopKitUI/Views/Settings Editors/BasalRateScheduleEditor.swift:142]: Hey @novalegraReal quick, just wanted to make sure mlee was ok with the wording on this guardrail before we proceed to test? I saw Janet's comment in the card, but wasn't sure if it referred to this or not. TIA :)

Comment (rickpasetto) [reply LoopKitUI/Views/Settings Editors/BasalRateScheduleEditor.swift:142]: @ginnyyadav : Anna's (@novalegra ) last day was Friday.  While I think she's still got her email, I think she's backpacking right now.  I recommend following up with mlee directly.

**Pull request merged**